main.js

```diff
@@ -19,15 +19,19 @@ function Hoverable() {
 	var win = window;
 
 	function init() {
-		win.document.body.setAttribute('data-o-hoverable--js', '');
+		win.document.documentElement.setAttribute('data-o-hoverable--js', '');
 		touchSupport();
 	}
 
+	function updateClasses(action) {
```

Comment:
backwards compatibility. some people might be using e.g. `.o-js .o-hoverable-on` as opposed to `.o-js.o-hoverable-on`